cardano-testnet/test/cardano-testnet-test/files/golden/queries/govStateOut.json

```diff
@@ -660,9 +660,6 @@
         "txFeePerByte": 1,
         "utxoCostPerByte": 4310
     },
-    "futurePParams": {
```

Comment:
Any ideas why it shows up sometimes, but sometimes not in the query output?

Reply:
I am guessing it is something that depends on when in the epoch you are

Reply:
Ah, also, the issue is not that it sometimes doesn't show, but that the value is different. It says something like "unknown" sometimes. But I delete it with a patching function, to avoid the fluctuation. It may have to do with the fact that now I added a delay

Reply:
>  But I delete it with a patching function, to avoid the fluctuation. It may have to do with the fact that now I added a delay

This is unacceptable. Contact the ledger team and investigate.

Reply:
I've now asked in [#ledger](https://input-output-rnd.slack.com/archives/CCRB7BU8Y/p1722514920025959)